dace/transformation/interstate/loop_unroll.py

```diff
@@ -43,10 +43,12 @@ def can_be_applied(graph, candidate, expr_index, sdfg, strict=False):
             return False
         _, rng, _ = found
 
-        # If loop is not specialized or constant-sized, fail
-        if any(symbolic.issymbolic(r, sdfg.constants) for r in rng):
+        # If loop stride is not specialized or constant-sized, fail
+        if symbolic.issymbolic(rng[2], sdfg.constants):
+            return False
+        # If loop range diff is not constant-sized, fail
+        if symbolic.issymbolic(rng[1] - rng[0]):
```

Comment:
```suggestion
        if symbolic.issymbolic(rng[1] - rng[0], sdfg.constants):
```
Remember the constants!